SwiftWisdom/SwiftWisdom/Core/Foundation/Time/TimeOfDay.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  TimeOfDay.swift
+//  SwiftWisdom
+//
+//  Created by Logan Wright on 11/13/15.
+//  Copyright © 2015 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+/**
+ *  A struct represention of dates expressed as a time of day.
+ */
+public struct TimeOfDay {
+    
+    // MARK: Public Properties
+    
+    public let hours: Int
+    public let minutes: Int
+    
+    public var displayFormat: String = "h:mm a" // 9:18 am
```

Comment:
setting displayFormat might need to be reflected down into the private date formatter